.idea/gradle.xml

```diff
@@ -20,4 +21,4 @@
       </GradleProjectSettings>
     </option>
   </component>
-</project>
+</project>
```

Comment:
Not entirely sure what this does, but I suspect we want to separate out IDEA config changes and code changes.

Reply:
Hm, and though I left this comment on one line, I meant it for more the entirety of the changes to this file.

Reply:
Yes, this was introduced by accident.